werkzeug/_compat.py

```diff
@@ -151,10 +149,13 @@ def reraise(tp, value, tb=None):
     from io import StringIO, BytesIO
     NativeStringIO = StringIO
 
+    _latin1_decode = operator.methodcaller('decode', 'latin1')
```

Comment:
I don't like this particular refactoring, it makes the function bodies using it harder to read.